src/poke.sol

```diff
@@ -8,44 +10,45 @@ contract PipLike {
     function peek() public returns (bytes32, bool);
 }
 
-contract Spotter {
+contract Spotter is DSMath {
     PitLike public pit;
-    bytes32 public ilk;
-    PipLike public pip;
-    uint256 public mat;
+    mapping (bytes32 => Ilk) public ilks;
+    uint256 public par = RAY; // usd per dai
```

Comment:
It's actually "ref per dai", with the price feeds provided in "ref per gem". It just happens that we're choosing ref = usd at the outset.

Reply:
You are right. Fixed